templates/cli/lib/commands/push.js.twig

```diff
@@ -666,204 +793,154 @@ const pushCollection = async ({ all, yes } = {}) => {
                 collectionId: collection['$id'],
                 parseOutput: false,
-            })
-
-            log(`Collection ${collection.name} ( ${collection['$id']} ) already exists.`);
-
-            if (!yes) {
-                const answers = await inquirer.prompt(questionsPushCollections[1])
-                if (answers.override.toLowerCase() !== "yes") {
-                    log(`Received "${answers.override}". Skipping ${collection.name} ( ${collection['$id']} )`);
-                    continue;
-                }
-            }
-
-            log(`Deleting indexes and attributes ... `);
-
-            const { indexes } = await paginate(databasesListIndexes, {
-                databaseId,
-                collectionId: collection['$id'],
-                parseOutput: false
-            }, 100, 'indexes');
-
-            await Promise.all(indexes.map(async index => {
-                await databasesDeleteIndex({
-                    databaseId,
-                    collectionId: collection['$id'],
-                    key: index.key,
-                    parseOutput: false
-                });
-            }));
-
-            let result = await awaitPools.wipeIndexes(databaseId, collection['$id']);
-            if (!result) {
-                throw new Error("Index deletion timed out.");
-            }
-
-            const { attributes } = await paginate(databasesListAttributes, {
-                databaseId,
-                collectionId: collection['$id'],
-                parseOutput: false
-            }, 100, 'attributes');
+            });
 
-            await Promise.all(attributes.map(async attribute => {
-                await databasesDeleteAttribute({
-                    databaseId,
+            if (remoteCollection.name !== collection.name) {
+                await databasesUpdateCollection({
+                    databaseId: collection['databaseId'],
                     collectionId: collection['$id'],
-                    key: attribute.key,
+                    name: collection.name,
                     parseOutput: false
-                });
-            }));
+                })
 
-            const deleteAttributesPoolStatus = await awaitPools.wipeAttributes(databaseId, collection['$id']);
-            if (!deleteAttributesPoolStatus) {
-                throw new Error("Attribute deletion timed out.");
+                success(`Updated ${collection.name} ( ${collection['$id']} ) name`);
             }
+            collection.remoteVersion = remoteCollection;
 
-            await databasesUpdateCollection({
-                databaseId,
-                collectionId: collection['$id'],
-                name: collection.name,
-                documentSecurity: collection.documentSecurity,
-                permissions: collection['$permissions'],
-                enabled: collection.enabled,
-                parseOutput: false
-            })
+            collection.isExisted = true;
```

Comment:
What is the function of this attribute `isExisted` . Is this saved in the appwrite.json file?

Reply:
This is just for the local flow as the collection is no longer saved to the `appwrite.json` file